WcaOnRails/config/locales/en.yml

```diff
@@ -602,6 +602,15 @@ en:
       success: "Updated locale."
       failure: "Unable to update your profile."
       unavailable: "Requested locale is not available."
+    #context: user creating new account and receiving confirmation mail
+    mailer:
+      create_new_account:
+        header: "Confirmation instructions"
+        welcome: "Welcome %{user_email}!"
+        confirm: "Your account is created and must be activated before you can use it. Please confirm your account through the link below: "
+        activate: "After activation you may login to the website of the World Cube Association: "
+        addendum: "This email is generated automatically. If you didn't register an account on the website of the World Cube Association and received this message by mistake, please ignore and delete it."
+        questions: "Do you have more questions? We will be happy to answer them! Feel free to send a mail to "
```

Comment:
Translations like this are always concerning to me, because it assumes that the language is going to want this link or whatever to come at the end of the sentence. We prefer to use string interpolation whenever possible, so this would become something like:

```
questions: "Do you have more questions? We will be happy to answer them! Feel free to send a mail to %{contact_link}."
```

After this change, you would also have to change view code a bit.

I think you could make this same sort of change for the `activate` rule above.